prov/efa/src/rdm/rxr_pkt_cmd.c

```diff
@@ -65,111 +65,161 @@
  * @related rxr_pkt_entry
  */
 static
-int rxr_pkt_entry_fill_wiredata(struct rxr_pkt_entry *pkt_entry,
-				int pkt_type,
-				struct efa_rdm_ope *ope)
+int rxr_pkt_fill_wiredata(struct rxr_pkt_entry *pkt_entry,
+			  int pkt_type,
+			  struct efa_rdm_ope *ope,
+			  int64_t data_offset,
+			  int data_size)
```

Comment:
can you add a docstring for this function?

Reply:
There is already a docstring. I updated it to make it fit